src/sqs/mod.rs

```diff
@@ -91,7 +91,9 @@ pub struct SqsMessageObj<T: Serialize> {
 pub struct SqsMessageAttribute {
     pub string_value: Option<String>,
     pub binary_value: Option<Base64Data>,
+    #[serde(default)]
     pub string_list_values: Vec<String>,
+    #[serde(default)]
```

Comment:
While `serde(default)` fixes the issue here, it also makes the absence of a value here undistinguishable from an empty array.
So, perhaps `Option<Vec<String>>` would be a better choice here.

But we should also keep in mind that those two fields are not actually used for anything yet and just reserved for future use (see [AWS docs here](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_MessageAttributeValue.html)).
So, it’s hard to tell now whether this limitation may cause any problems in the future or now.